### PR TITLE
about directory set up

### PR DESCRIPTION
**Pull request by JaniceHe26**
About.jsx and About.css

### CODE REVIEW

File: src/components/about/About.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (wizeone) [inline src/components/about/About.jsx]:
```suggestion
import React from 'react';
```

File: src/components/about/About.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+export default function About(){
```
Comment (wizeone) [inline src/components/about/About.jsx:3]: What do you think about moving 'export default' to the bottom of the page similar to the defaull App.js structure? It's general practice to have the exporting separate from the function declaration.

File: src/components/about/About.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  return (
+    <React.Fragment>
+    <h4 className="aboutTitle">About</h4>
+
+    <div className="aboutDiv">
+    <img className="aboutImg" src="" alt="" title=""></img>
+    <p className="aboutText"></p>
+    
+    </div>
+    </React.Fragment>
```
Comment (wizeone) [inline src/components/about/About.jsx]:
```suggestion
    <React.Fragment>
      <h4 className="aboutTitle">About</h4>

      <div className="aboutDiv">
        <img className="aboutImg" src="" alt="" title=""></img>
        <p className="aboutText"></p>
    
      </div>
    </React.Fragment>
```

**Review by wizeone — CHANGES_REQUESTED** [review]

File: src/components/about/About.jsx (hunk 2 above)
Comment (JaniceHe26) [reply src/components/about/About.jsx:3]: for sure!

**Pull request merged**